composer.json

```diff
@@ -3,6 +3,7 @@
 	"type": "typo3-cms-extension",
 	"description": "A low-level tool to backup and restore a full TYPO3 installation",
 	"license": ["GPL-2.0+"],
+	"version": "1.0.0"
```

Comment:
In ext_emconf.php this is already there, closing this pull request. Then a tag can be created for this extension.